packages/guess-webpack/README.md

```diff
@@ -19,7 +19,7 @@ const { GuessPlugin } = require('guess-webpack');
 Add this to the end of your webpack config:
 
 ```js
-new GuessPlugin({ GA: 'GOOGLE_ANALYTICS_VIEW_ID' });
+new GuessPlugin({ GA: 'GOOGLE_ANALYTICS_VIEW_ID', jwt: credentials});
```

Comment:
It's not yet necessary for everyone. Google hasn't seemed to have enforced this strict check everywhere just yet. Would you mention what are the preconditions (i.e. if you can't authenticate via OAuth2 you will have to specify a jwt)?

Reply:
I can, though OAuth2 doesn't work for me.
```
if (!c.jwt) {
        client = google_oauth2_node_1.auth({
            clientId: clientId,
            clientSecret: clientSecret,
            scope: scope
        }).then(function (token) {
            var oauth2Client = new google.auth.OAuth2();
            oauth2Client.setCredentials(token);
            return oauth2Client;
        });
    }
    else {
        client = Promise.resolve(new google.auth.JWT(c.jwt.client_email, null, c.jwt.private_key, [scope], null));
    }
```
Only the else statement works.